### ISSUE TITLE
Create function to stop r5_core

### ISSUE THREAD

**Issue by dhersz** [ipeaGIT/r5r#16]
I tried stopping the process with `system("Taskkill /IM java.exe /F", intern = TRUE)` but wasn't able to.

For reference, when I logoff my computer after setting r5 up i get the following warnings:

```r
ago 15, 2020 3:16:35 PM org.geotools.referencing.factory.DeferredAuthorityFactory disposeBackingStore
INFO: Disposing class org.geotools.referencing.factory.epsg.hsql.ThreadedHsqlEpsgFactory backing store
ago 15, 2020 3:16:35 PM org.hsqldb.persist.Logger logInfoEvent
INFO: Database closed
```

Actually I don't even know what task I should be trying to kill here, I tried java because that's how you'd stop a OTP server, but not sure how it works with r5.

**Comment by mvpsaraiva** [comment]
The r5_core does not work the same way as the OTP server. Technically, if you delete the r5_core object from memory, it should kill the whole process. This should work:

`rm(r5_core)`

When you close RStudio, or restart your R session, r5_core should stop also.

**Comment by dhersz** [comment]
Done.

**Comment by rafapereirabr** [comment]
I've tryind using `rm(r5_core)` while running `travel_time_matrix` and it did not stop. Daniel, did this work for you?

**Comment by dhersz** [comment]
I didn't try that, but I guess that `rm`ing after starting to build the matrix doesn't affect anything, since what matters is the `r5_core` being alive in the moment of the function call. Does that make sense?

**Comment by mvpsaraiva** [comment]
@rafapereirabr How did you manage to call `rm(r5_core)` while running `travel_time_matrix`? Shouldn't R console be stuck during execution?
Anyways, I don't think removing `r5_core` from R environment would "hard-stop" it's execution...

**Comment by rafapereirabr** [comment]
> @rafapereirabr How did you manage to call `rm(r5_core)` while running `travel_time_matrix`? Shouldn't R console be stuck during execution?
> Anyways, I don't think removing `r5_core` from R environment would "hard-stop" it's execution...

Yep, I hopelesly tried to call `rm(r5_core)` while running `travel_time_matrix`
